nixos/lib/make-system-tarball.sh

```diff
@@ -54,8 +53,8 @@ mkdir -p $out/tarball
 
 rm env-vars
 
-tar --sort=name --mtime='@1' --owner=0 --group=0 --numeric-owner -cvJf $out/tarball/$fileName.tar.xz * $extraArgs
+time tar --sort=name --mtime='@1' --owner=0 --group=0 --numeric-owner -c * $extraArgs | $compression $compressorFlags > $out/tarball/$fileName.tar.${extension}
```

Comment:
did you intend to leave time in here or was it just for testing?

Reply:
a bit of both, with the `-v` gone its a bit too silent, and it could be handy to know how long just the compression phase took, since half the time is taken up by a `cp`